h2o-core/src/main/java/water/network/CertAndKeyGenWrapper.java

```diff
@@ -0,0 +1,40 @@
+package water.network;
+
+import sun.security.x509.X500Name;
```

Comment:
OpenJDK should be ok (http://grepcode.com/file/repository.grepcode.com/java/root/jdk/openjdk/7u40-b43/sun/security/x509/X500Name.java#X500Name here's source for 7u40 but there are also sources for 6 and 8).

I'll have to take a look at IBM VM